apps/site/pages/docs/multiple-page-templates.mdx

```diff
@@ -0,0 +1,103 @@
+
+## PDP Template
+
+When rendering the PDPs, FastStore will use the product information to find the appropriate document from the CMS (template) to render, starting from the most specific template (slug) of the product to the most generic (`department` or the generic PDP).
+
+The order should be as follows:
+
+1.  A PDP document that matches the product slug;
+2.  A PDP document that matches the product `subcategory`;
+3.  A PDP document that matches the product `category`;
+4.  A PDP document that matches the product `department` (collection);
+5.  If no matches are found, use the generic PDP template.
+
+![PDP document list](https://vtexhelp.vtexassets.com/assets/docs/src/pdplist___a4862453821a4f753ac1d171cc3708b6.png)
+
+Let's see the examples below:
+
+While using PDPs, note that this type of page is recognized by the slug pattern [https://starter.vtex.app/apple-magic-mouse/p](https://starter.vtex.app/apple-magic-mouse/p)
+
+You can create the template based on the slug `/apple-magic-mouse/p`.
+
+![PDP Slug template](https://vtexhelp.vtexassets.com/assets/docs/src/pdpslug___c271bf0ba13f048deee0da6914dbc601.png)
+
+This is the most specific template, when the slug template exists, it will be rendered.
+But there are cases where you would like to render a specific template for products inside a `subcategory` or a `department`, for instance, instead of using the slug.
+Because of that, the template can assume different behaviors based on the input value beyond the slug: `subcategory`, `category`, `department`, or the generic template.
+
+The products in VTEX catalog are under a `category` tree, usually `/department/category/subcategory/`.
+Note that the product mentioned above is in the Technology `department` from the catalog. Then, you can create a `department` template.
+
+![PDP Department template](https://vtexhelp.vtexassets.com/assets/docs/src/pdpdepartment___402731c744a6ad033442eefabe442875.png)
+
+If this product were under a `subcategory`, would also be possible to create a template representing this scenario. The category tree is used in the template input, separated by slashes.
+
+![PDP subcategory template](https://vtexhelp.vtexassets.com/assets/docs/src/pdpsubcategory___902f100603227e1609763f96600a9e0f.png)
+
+You can also see the category hierarchy through the breadcrumb.
+
+![PDP breadcrumb](https://vtexhelp.vtexassets.com/assets/docs/src/breadcrumb___c9c12da51f685572dfb3a9403168b26a.png)
+
+If there are no matches for slug or `department`, the generic template will be used to render this PDP. So you need to create a document with an empty value in the template input field.
+
+![PDP general template](https://vtexhelp.vtexassets.com/assets/docs/src/pdpgeneral___374a4398e4c35e5374c09c07794498dc.png)
+
+Possible templates in CMS from this product:
+
+- Slug: `/apple-magic-mouse/p`
+- Department: `/technology/`
+- PDP generic template: (empty value)
+
+### PLP Template
```

Comment:
Which are the available fields in `settings` for this type? Do you know if I can add new ones or change them?
I was checking [this example](https://vendemo.myvtex.com/admin/new-cms/faststore/plp/edit/1c666277-8201-4f07-97c6-111931790da5), and I was curious to know if other fields could be implemented.